ReactiveCocoa/UIKit/UIGestureRecognizer.swift

```diff
@@ -0,0 +1,25 @@
+import ReactiveSwift
+import UIKit
+import enum Result.NoError
+
+extension Reactive where Base: UIGestureRecognizer {
+	
+	/// Create a signal which sends a `next` event for each gesture event
+	///
+	/// - returns:
+	///   A trigger signal.
+	
+	public var recognised: Signal<(), NoError> {
+		return Signal { observer in
+			let receiver = CocoaTarget(observer)
+			base.addTarget(receiver, action: #selector(receiver.sendNext))
```

Comment:
Ooooohhh. I thought - with my aged memory in UIKit - the action is just for getting the final ended/recognised event. It turns out to be not. In this case, `trigger` is the right name. I am sorry about this. Would you mind to change it back?

🙏